doc/SpectatorView.Native.md

```diff
@@ -1 +1,77 @@
-Coming soon...
+# SpectatorView.Native Dlls
+
+The following Dlls are built out of the [SpectatorView.Native.sln](../src/SpectatorView.Native/SpectatorView.Native.sln).
+
+- **SpectatorView.Compositor.dll** is needed for DSLR camera calibration and DSLR camera spectating experiences.
+- **SpectatorView.Compositor.UnityPlugin.dll** is needed for DSLR camera calibartion and DSLR camera spectating experiences.
+- **SpectatorView.OpenCV.dll** is used in both ArUco marker detection and DSLR camera calibration.
+
+# SpectatorView.Compositor.dll & SpectatorView.Compositor.UnityPlugin.dll
+
+### 1. Obtain external dependencies
+**DeckLink Capture Card**
+If you are using a Blackmagic capture card, you will need to install the SDK and create a Visual Studio user macro for its location.
++ Download the DeckLink SDK from here: https://www.blackmagicdesign.com/support - Search for Desktop Video SDK in "Latest Downloads"
++ Extract the SDK anywhere on your computer.
++ Update the DeckLink_inc user macro in [dependencies.props](../src/SpectatorView.Native/SpectatorView.Compositor/dependencies.props) with the corresponding path on your computer.

```

Comment:
>Update the DeckLink_inc user macro in [dependencies.props](../src/SpectatorView.Native/SpectatorView.Compositor/dependencies.props) with the corresponding path on your computer. [](start = 2, length = 177)

One of the changes we made internally was to add a dependencies.users.props that dependencies.props conditionally loads. The rationale was that you could have dependencies.users.props in a .gitignore and then not have to deal with changing a file that's actually checked into Git.  What do you think about documenting that workflow or keeping it?

Reply:
this seems worth adding but may be beyond the scope of this review. I've opened https://github.com/microsoft/MixedReality-SpectatorView/issues/51 to address this concern.